src/main/java/org/jitsi/jicofo/MeetExtensionsHandler.java

```diff
@@ -131,26 +131,13 @@ public void processPacket(Packet packet)
             return;
         }
 
-        if (packet instanceof ColibriConferenceIQ)
-        {
-            handleColibriIq((ColibriConferenceIQ) packet);
-        }
-        else if (packet instanceof MuteIq)
-        {
-            handleMuteIq((MuteIq) packet);
-        }
-        else if (packet instanceof RayoIqProvider.DialIq)
-        {
-            handleRayoIQ((RayoIqProvider.DialIq) packet);
```

Comment:
I would go with utility class instead of duplicating the code:
https://gist.github.com/paweldomas/dff5be0aa4e1024b3d51daeb94d52eda